index.js

```diff
@@ -1,3 +1,4 @@
+import {AsyncResource} from 'node:async_hooks';
```

Comment:
This package is used in the browser a lot too, so you cannot directly call Node.js only APIs.

You could create a small wrapper for asyncresource.bind, put it in a file, make another placeholder one that does nothing, and then use subpath imports to delegate the placeholder one to browser and the real one to Node.js: https://nodejs.org/api/packages.html#subpath-imports

Reply:
Like so? https://github.com/sindresorhus/p-limit/pull/71/commits/9543ab0fc3e48e972a9e9bc5ad3928e21e16ed24

Reply:
Yes, exactly 👍

Reply:
Thanks!